js_modules/dagster-ui/packages/ui-core/src/assets/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -694,7 +694,10 @@ const UpstreamUnavailableWarning = ({
     upstreamAssetHealth.some((a) => {
       // If the key is not undefined, it's present in the partition key space of the asset
       return (
-        a.dimensions.length && a.stateForKey([singleDimensionKey]) === AssetPartitionStatus.MISSING
+        a.dimensions.length &&
+        ![AssetPartitionStatus.MATERIALIZED, AssetPartitionStatus.MATERIALIZING].includes(
+          a.stateForKey([singleDimensionKey]),
+        )
```

Comment:
Stylistic nit, but I find this pretty hard to understand with the negation of the `includes`. I think it would be clearer as:

```
const stateForKey = a.stateForKey([singleDimensionKey]);
return (
  a.dimensions.length &&
  stateForKey !== AssetPartitionStatus.MATERIALIZED &&
  stateForKey !== AssetPartitionStatus.MATERIALIZING
);
```

I'm actually confused enough that I'm not totally sure my suggested snippet here is equivalent... 😆

Reply:
Ahh think that's equivalent, will change this so it's clearer 👍